src/views/DashPhotographer/ClientActionsMenu.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div class="text-right">
+    <v-menu bottom left>
+      <template v-slot:activator="{ on, attrs }">
+        <v-btn dark icon v-bind="attrs" v-on="on">
+          <v-icon color="primary">mdi-dots-vertical</v-icon>
+        </v-btn>
+      </template>
+      <v-list>
+        <v-list-item
+          v-for="(action, i) in actions"
+          :key="i"
+          @click="action.action(client)"
+        >
+          <component
+            :is="action.component"
+            v-if="action.component"
+            :client="client"
+          />
+          <v-list-item-title v-else>
+            <v-icon class="mr-2" :color="action.color">
+              {{ action.icon }}
+            </v-icon>
+            <span :class="`${action.color}--text`">
+              {{ action.title }}
+            </span>
+          </v-list-item-title>
+        </v-list-item>
+      </v-list>
+    </v-menu>
+  </div>
+</template>
+
+
+
+
+<script>
+import { navigateTo } from "@/utils/utils";
+import DialogEditClient from "./DialogEditClient";
+import DialogArchiveClient from "./DialogArchiveClient";
+export default {
+  props: ["client"],
+  data() {
+    return {
+      actions: [
+        {
+          action: () => {}, // unneeded, empty action
```

Comment:
I dont like the function place holder. somehow the @click and the v-for should be seperated in:
     
```
 <v-list-item
      v-for="(action, i) in actions"
      :key="i"
      @click="action.action(client)" >
```